stable/aws-efa-k8s-device-plugin/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: aws-efa-k8s-device-plugin
 description: A Helm chart for EFA device plugin.
-version: v0.5.4
-appVersion: "v0.5.4"
+version: v0.5.5
+appVersion: "v0.5.5"
```

Comment:
appVersion should stay at "v0.5.4" which is the latest image version ?

Reply:
Yes, let's keep appVersion same as the efa-plugin version so that we can keep a track

Reply:
ah yes - makes sense. corrected in https://github.com/aws/eks-charts/pull/1151/commits/7f82abe15fd15068ccbf8724b0fa4a353965c4e4